### ISSUE TITLE
Node scaling (label scaling)

### ISSUE THREAD

**Issue by carlskii** [almende/vis#1458]
Hi,

I've created a group for my nodes like this:

```javascript
cm: {
                color: "#FFA807",
                shape: 'circle',
                "borderWidth": 3,
                size: 200,
                font: {size:20, face:'arial'}
            }
```
However, the node always seems to scale based on the length of the text contained within the node. Is it possible to fix the size of the node and truncate the text that spills over ?

**Comment by AlexDM0** [comment]
Hi,

Nodes with text inside are scales using the font size if you enable it. Size not affect it as stated in the docs. 

Cheers

**Comment by carlskii** [comment]
OK I'll give this a try.

**Comment by carlskii** [comment]
OK so I tried this: 

```javascript
cm: {
                color: "#FFA807",
                shape: 'circle',
                fontSize: 10
            }
```

However my nodes still seem to scale up/down based on the character lengths.

**Comment by AlexDM0** [comment]
Take a look at the scaling option. Please read the documentation.

**Comment by AlexDM0** [comment]
My bad, the scaling of labels is currently broken and will be fixed with the release of tomorrow.

Regards

**Comment by AlexDM0** [comment]
This has been fixed in 4.10
